src/onefetch/deps/package_parser.rs

```diff
@@ -0,0 +1,26 @@
+use crate::onefetch::error::*;
+use {regex::Regex, toml::Value};
+
+pub fn cargo(contents: &str) -> Result<usize> {
+    let parsed = contents.parse::<Value>()?;
+
+    Ok(parsed["dependencies"].as_table().unwrap().len())
```

Comment:
A `Cargo.toml` file is valid even if it doesn't have a `[dependencies]` key, so could you use [`toml::Value::get`](https://docs.rs/toml/0.5.7/toml/value/enum.Value.html#method.get), returning 0 for the count if `None` is returned?

Reply:
Oh yeah, good point. I didn't think of that.

Reply:
1d6d95b